src/renderer/atlas/BuiltinGlyphs.cpp

```diff
@@ -0,0 +1,1276 @@
+    {
+        Instruction{ Shape_ClosedFilledPath, Pos_1_1, Pos_0_1, Pos_0_1, Pos_1_2 },
+        Instruction{ Shape_ClosedFilledPath, Pos_1_1, Pos_1_1, Pos_Min, Pos_Min },
+    },
+    // U+E0B3 Left triangle line
+    {
+        Instruction{ Shape_OpenLinePath, Pos_1_1, Pos_0_1, Pos_0_1_Add_0_5, Pos_1_2 },
+        Instruction{ Shape_OpenLinePath, Pos_1_1, Pos_1_1, Pos_Min, Pos_Min },
+    },
+    // U+E0B4 Right semi-circle solid
+    {
+        Instruction{ Shape_FilledEllipsis, Pos_0_1, Pos_1_2, Pos_1_1, Pos_1_2 },
+    },
+    // U+E0B5 Right semi-circle line
+    {
+        Instruction{ Shape_EmptyEllipsis, Pos_0_1, Pos_1_2, Pos_1_1_Sub_0_5, Pos_1_2_Sub_0_5 },
+    },
+    // U+E0B6 Left semi-circle solid
+    {
+        Instruction{ Shape_FilledEllipsis, Pos_1_1, Pos_1_2, Pos_1_1, Pos_1_2 },
+    },
+    // U+E0B7 Left semi-circle line
+    {
+        Instruction{ Shape_EmptyEllipsis, Pos_1_1, Pos_1_2, Pos_1_1_Sub_0_5, Pos_1_2_Sub_0_5 },
+    },
+    // U+E0B8 Lower left triangle
+    {
+        Instruction{ Shape_ClosedFilledPath, Pos_0_1, Pos_0_1, Pos_0_1, Pos_1_1 },
+        Instruction{ Shape_ClosedFilledPath, Pos_1_1, Pos_1_1, Pos_Min, Pos_Min },
+    },
+    // U+E0B9 Backslash separator
+    {
+        Instruction{ Shape_LightLine, Pos_0_1, Pos_0_1, Pos_1_1, Pos_1_1 },
+    },
+    // U+E0BA Lower right triangle
+    {
+        Instruction{ Shape_ClosedFilledPath, Pos_0_1, Pos_1_1, Pos_1_1, Pos_1_1 },
+        Instruction{ Shape_ClosedFilledPath, Pos_1_1, Pos_0_1, Pos_Min, Pos_Min },
+    },
+    // U+E0BB Forward slash separator
+    {
+        Instruction{ Shape_LightLine, Pos_0_1, Pos_1_1, Pos_1_1, Pos_0_1 },
+    },
+    // U+E0BC Upper left triangle
+    {
+        Instruction{ Shape_ClosedFilledPath, Pos_0_1, Pos_1_1, Pos_0_1, Pos_0_1 },
+        Instruction{ Shape_ClosedFilledPath, Pos_1_1, Pos_0_1, Pos_Min, Pos_Min },
+    },
+    // U+E0BD Forward slash separator
+    {
+        Instruction{ Shape_LightLine, Pos_0_1, Pos_1_1, Pos_1_1, Pos_0_1 },
+    },
+    // U+E0BE Upper right triangle
+    {
+        Instruction{ Shape_ClosedFilledPath, Pos_0_1, Pos_0_1, Pos_1_1, Pos_0_1 },
+        Instruction{ Shape_ClosedFilledPath, Pos_1_1, Pos_1_1, Pos_Min, Pos_Min },
+    },
+    // U+E0BF Backslash separator
+    {
+        Instruction{ Shape_LightLine, Pos_0_1, Pos_0_1, Pos_1_1, Pos_1_1 },
+    },
+};
+
+constexpr bool BoxDrawing_IsMapped(char32_t codepoint)
+{
+    return codepoint >= BoxDrawing_FirstChar && codepoint < (BoxDrawing_FirstChar + BoxDrawing_CharCount);
+}
+
+constexpr bool Powerline_IsMapped(char32_t codepoint)
+{
+    return codepoint >= Powerline_FirstChar && codepoint < (Powerline_FirstChar + Powerline_CharCount);
+}
+
+// How should I make this constexpr == inline, if it's an external symbol? Bad compiler!
+#pragma warning(suppress : 26497) // You can attempt to make '...' constexpr unless it contains any undefined behavior (f.4).
+bool BuiltinGlyphs::IsCustomGlyph(char32_t codepoint) noexcept
```

Comment:
I feel like I saw calls from AtlasEngine.cpp to both `xxx_IsMapped` and wondered why they didnt use a common function for it. Now I learn there *is* a common function for it!

Reply:
Those `IsMapped` calls are now gone and replaced with a call to this function.